### PR TITLE
[FEATURE] Create a custom model for a Core table

### PR DESCRIPTION
**Pull request by brotkrueml**
Releases: main, 11.5

### CODE REVIEW

File: Documentation/ExtensionArchitecture/Extbase/Reference/Domain/_Content.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyVendor\MyExtension\Domain\Model;
+
+use TYPO3\CMS\Extbase\DomainObject\AbstractEntity;
+
+class Content extends AbstractEntity
```
Comment (linawolf) [inline Documentation/ExtensionArchitecture/Extbase/Reference/Domain/_Content.php:9]: why is this class not final?

File: Documentation/ExtensionArchitecture/Extbase/Reference/Domain/_ContentRepository.php [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyVendor\MyExtension\Domain\Repository;
+
+use TYPO3\CMS\Extbase\Persistence\Repository;
+
+class ContentRepository extends Repository
```
Comment (linawolf) [inline Documentation/ExtensionArchitecture/Extbase/Reference/Domain/_ContentRepository.php:9]: shouldn't this be final?

File: Documentation/ExtensionArchitecture/Extbase/Reference/Domain/_Content.php (hunk 1 above)
Comment (brotkrueml) [reply Documentation/ExtensionArchitecture/Extbase/Reference/Domain/_Content.php:9]: Hmm, thought about it. But not sure as we should propose models as final. But as you have the same opinion, okay,

Comment (linawolf) [reply Documentation/ExtensionArchitecture/Extbase/Reference/Domain/_Content.php:9]: Mmh that is true, models are probably something you always want to be able to extend

**Review by linawolf — APPROVED** [review]

**Pull request merged**